SPECS/php/php.spec

```diff
@@ -36,22 +30,27 @@
 %global with_sodium   1
 %global with_pspell   1
 %global with_lmdb     1
-
+%global with_db4      1
+%global with_tidy     1
+%global with_qdbm     0
+%global upver        8.1.11
```

Comment:
Prefer to remove this macro too.  The "Version:" version should be enough, is easier to see and I believe it can be referenced in other places in the file as %{version}